dev/scripts/docs-to-google-sheets.js

```diff
@@ -0,0 +1,126 @@
+const path = require('path');
```

Comment:
Is this script under control of our codestyle? Does lint work for scripts? How can we use magic numbers, for instance?